picoquic/frames.c

```diff
@@ -277,6 +277,45 @@ int picoquic_decode_stream_reset_frame(picoquic_cnx_t* cnx, uint8_t* bytes,
     return ret;
 }
 
+/*
+ * New Connection ID frame
+ */
+
+int picoquic_skip_connection_id_frame(uint8_t* bytes, size_t bytes_max, size_t* consumed) 
+{
+    int ret = 0;
+    int cid_length;
+    size_t byte_index = 1;
+
+    byte_index += picoquic_varint_skip(bytes + byte_index);
+    if (byte_index >= bytes_max) {
+        cid_length = 0;
+    } else {
+        cid_length = bytes[byte_index++];
+    }
+    byte_index += cid_length + 16;
+
+    if (byte_index < bytes_max) {
+        *consumed = byte_index;
+    } else {
+        *consumed = bytes_max;
+        ret = -1;
+    }
+
+    return ret;
+}
+
+int picoquic_decode_connection_id_frame(picoquic_cnx_t* cnx, uint8_t* bytes, size_t bytes_max, size_t* consumed)
+{
+    int ret = picoquic_skip_connection_id_frame(bytes, bytes_max, consumed);
```

Comment:
Why are you skipping the frame? Should that just be a first implementation? Maybe I would add a `TODO: Store new ID `

Reply:
Yes.

Reply:
Yes.